src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -184,11 +184,13 @@ HARDWARE_INTRINSIC(Sve,           SaturatingIncrementBy32BitElementCount,
 HARDWARE_INTRINSIC(Sve,           SaturatingIncrementBy64BitElementCount,                           -1,      3,      true,  {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_sqincd,     INS_sve_uqincd,     INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_HasImmediateOperand|HW_Flag_HasEnumOperand|HW_Flag_SpecialCodeGen|HW_Flag_HasScalarInputVariant|HW_Flag_SpecialImport|HW_Flag_HasRMWSemantics)
 HARDWARE_INTRINSIC(Sve,           SaturatingIncrementBy8BitElementCount,                             0,      3,      false, {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_sqincb,     INS_sve_uqincb,     INS_sve_sqincb,     INS_sve_uqincb,     INS_invalid,        INS_invalid},     HW_Category_Scalar,                HW_Flag_Scalable|HW_Flag_HasImmediateOperand|HW_Flag_HasEnumOperand|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_HasRMWSemantics)
 HARDWARE_INTRINSIC(Sve,           SaturatingIncrementByActiveElementCount,                          -1,      2,      true,  {INS_invalid,        INS_sve_sqincp,     INS_sve_sqincp,     INS_sve_sqincp,     INS_sve_sqincp,     INS_sve_sqincp,     INS_sve_sqincp,     INS_sve_sqincp,     INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_BaseTypeFromSecondArg|HW_Flag_HasRMWSemantics)
+HARDWARE_INTRINSIC(Sve,           Scale,                                                            -1,      -1,     false, {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_fscale,     INS_sve_fscale},  HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_BaseTypeFromFirstArg|HW_Flag_EmbeddedMaskedOperation|HW_Flag_LowMaskedOperation|HW_Flag_HasRMWSemantics)
```

Comment:
Don't think we need `HW_Flag_BaseTypeFromFirstArg` for this one. Can you double check?